component-interaction/src/app/pages/user.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from "@angular/core";
+import { ActivatedRoute } from "@angular/router";
+
+import { UserModel } from "../models/user.model";
+import { UserService } from "../services/user.service";
+
+@Component({
+	selector: "user",
+	templateUrl: "./user.component.html",
+	styleUrls: ["./user.component.css"]
+})
+export class UserComponent implements OnInit {
+	public constructor(private userService: UserService, private route: ActivatedRoute) { }
+	public user: UserModel = {
```

Comment:
```suggestion
	public user?: UserModel;
```

and move this class property above constructor